.github/workflows/deploy_prod.yml

```diff
@@ -18,13 +18,38 @@ jobs:
       ecs-cluster-name: "isomer-prod-ecs"
       ecs-web-service-name: "isomer-prod-ecs-service"
       ecs-container-name: "backend"
+      ecs-container-port: 8081
       environment: "prod"
       shortEnv: "prod"
       task-definition-path: ".aws/deploy/backend-task-definition.prod.json" 
       codedeploy-application: "isomer-prod-ecs-app"
       codedeploy-deployment-group: "isomer-prod-ecs-dg"
+      path-to-dockerfile: "Dockerfile"
 
     secrets: 
       AWS_ACCOUNT_ID: ${{ secrets.AWS_ACCOUNT_ID }}
       EFS_FILE_SYSTEM_ID: ${{ secrets.PROD_EFS_FILE_SYSTEM_ID }}
       DD_API_KEY: ${{ secrets.DD_API_KEY }}
+
+  deploy-support: 
+    name: Deploy support to staging
+    uses: ./.github/workflows/aws_deploy.yml
+    with:
+      aws-region: "ap-southeast-1"
+      cicd-role: "arn:aws:iam::095733531422:role/isomer-infra-github-oidc-role-16ea937"
+      ecr-repository: "isomer-infra-prod-ecr"
+      ecs-cluster-name: "isomer-prod-ecs"
+      ecs-web-service-name: "prod-support-ecs-service"
```

Comment:
The naming convention is not consistent to the name for deploy-app above (`isomer-prod-ecs-service`). Did you mean `isomer-support-ecs-service` instead?

Reply:
no, this is very annoying and an oversight on my part! 

first, there's a strict upper limit on the number of characters we can use for the `name` here - this is because of pulumi (it respects AWS' [length conventions](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-elasticloadbalancingv2-targetgroup.html#cfn-elasticloadbalancingv2-targetgroup-name) for load balancer target groups) 

second, because of the above, i opted to go with `isomer-<env>` for app last time (i didn't know there would be a separate ecs service...). hence now, when i ran into the same length issue, i opted to go with `<env>-<service>`

Reply:
sadly enough, this leads to it being confusing...